cirrus/google_cloud/iam.py

```diff
@@ -63,6 +68,23 @@ def __str__(self):
 
         return str(output_dict)
 
+    def get_dict(self):
+        """
+        Return representation of object
+
+        Returns:
+            str: Representation of the Policy which can be POSTed to Google's API
+        """
+        output_dict = dict()
+        output_dict["policy"] = dict()
+        output_dict["policy"]["bindings"] = [
```

Comment:
you could just do
```
return {
  "policy": {
    "bindings": ...
    "etag": ...
    "version": ...
  }
}
```

Reply:
yeah, for consistency I was matching the existing function

Reply:
oh i see, then maybe
- simplify `GooglePolicy.get_dict`
- make `GooglePolicy.__str__()` call `GooglePolicy.get_dict`
- simplify `GooglePolicyBinding.get_dict` if you want to :p

Reply:
oh, :man_facepalming: yeah good point